ui/src/components/TaskGroupTable/index.jsx

```diff
@@ -47,7 +47,7 @@ const filterTasksByState = curry((filter, tasks) =>
 const filterTasksByName = curry((searchTerm, tasks) =>
   searchTerm
     ? tasks.filter(({ node: { metadata: { name } } }) =>
-        name.includes(searchTerm)
+        lowerCase(name).includes(searchTerm)
```

Comment:
where is `lowerCase` defined?

Reply:
https://github.com/taskcluster/taskcluster/blob/a1d7bb1c1270e5a0006545fb9ae83d33a62b4271/ui/src/components/TaskGroupTable/index.jsx#L4

Reply:
oh deer, I was sure I fully expanded the file when I did the search